portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -18,12 +18,18 @@
 public final class Comment {
 
   private final long id;
+  private final String user;
   private final String text;
   private final long timestamp;
+  private final long likes;
+  private final long dislikes;
 
-  public Comment(long id, String text, long timestamp) {
+  public Comment(long id, String user, String text, long timestamp, long likes, long dislikes) {
```

Comment:
No necessary change in this CL. But I'd recommend builder pattern to avoid constructor explosion.

https://www.geeksforgeeks.org/builder-design-pattern/

Reply:
Okay, I'll look into it!